circle.yml

```diff
@@ -14,8 +14,50 @@ jobs:
           name: "Build"
           command: cmake --build ~/build
 
+  test-docs:
+    docker:
+      - image: ethereum/cpp-build-env:experimental
+    steps:
+      - checkout
+      - run:
+          name: "Test documentation"
+          command: |
+            doxygen docs/Doxyfile > doxygen.log 2> doxygen.warnings
+            if [ -s doxygen.warnings ]; then
+              printf '\n\nDoxygen warnings:\n\n'
+              cat doxygen.warnings
+              exit 1
+            fi
+            cat doxygen.log
+
+  upload-docs:
+    docker:
+      - image: ethereum/cpp-build-env:experimental
+    steps:
+      - checkout
+      - run:
+          name: "Generate documentation"
+          command: doxygen docs/Doxyfile
+      - run:
+          name: "Upload documentation"
+          command: |
+            git config user.email "docs-bot@ethereum.org"
+            git config user.name "Documentation Bot"
+            git add docs
```

Comment:
Actually can you move it to the root instead so that https://ethereum.github.io/evmc works.

Reply:
I will try.

Reply:
Both work for me equally.